experiments/dataSources/Salesforce_Einstein_Analytics/readme.md

```diff
@@ -0,0 +1,113 @@
+## Salesforce Einstein Analytics data in Skuid
+Einstein Analytics is a powerful data integration and analysis platform.  It does a lot of things Skuid cannot do.  But there are going to be times when you would like to incorporate a “EA” report into your Skuid page and use Skuid components to show the data - rather than use an Einstein Dashboard or component.  Here is how. 
+
+Some resources. 
+
+- [Get a Free Developer Org](https://developer.salesforce.com/promotions/orgs/analytics-de) - with EA enabled,  and sample datasets (that we will be using in these pages)
+- [Ananlytics API doc](https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_rest_resources_overview.htm) (from Salesforce Developer resources):  
+- [Analytics query language (SAQL) reference](https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_saql.meta/bi_dev_guide_saql/bi_saql_intro.htm)
+
+
+## How to use this data source
+
+### Create the data source
```

Comment:
Since we're doing the same thing as the other Salesforce APIs listed here, why don't we have a Salesforce directory with a main README, and then have this as a subdirectory? Or cross-reference the setup steps instead of listing them in two places?